python/doc/user_manual/threshold_probability_simulation_algorithms.rst

```diff
@@ -23,6 +23,9 @@ Simulations methods
     SubsetSampling
     AdaptiveDirectionalStratification
     NAIS
+    experimental.CrossEntropyImportanceSampling
+    experimental.PhysicalSpaceCrossEntropyImportanceSampling
+    experimental.StandardSpaceCrossEntropyImportanceSampling
```

Comment:
the new classes must be added to the ChangeLog file

Reply:
this is done.